geomstats/geometry/hypersphere.py

```diff
@@ -173,6 +174,71 @@ def tangent_spherical_to_extrinsic(
 
         return tangent_vec_extrinsic
 
+    def extrinsic_to_spherical(self, point_extrinsic):
```

Comment:
Add a docstring?

It would be nice to add there which point is considered the origin of the spherical coordinates.

Reply:
Sorry forgot! Weird that DeepSource didn't detect it though

Reply:
Yeah....

Reply:
And now adding docstrings fails codecov?